src/mudlet.cpp

```diff
@@ -5036,3 +5036,16 @@ dlgTriggerEditor* mudlet::createMudletEditor()
 
     return pEditor;
 }
+
+bool mudlet::profileExists(const QString& profileName)
+{
+    const QStringList profiles = QDir(mudlet::getMudletPath(mudlet::profilesPath))
+                                 .entryList(QDir::Dirs | QDir::NoDotAndDotDot, QDir::Name);
+
+    if (profiles.contains(profileName)) {
```

Comment:
We can't be case sensitive here - not all file-systems that Mudlet might encounter are case-preserving/case-sensitive:
```suggestion
    if (profiles.contains(profileName, Qt::CaseInsensitive)) {
```